### PR TITLE
Add a checklist of administrator responsibilities

### PR DESCRIPTION
**Pull request by inoxx03**
@stoobie @cbudz @bergerhoffer @redhat-documentation/ccs-style-council Please feel free to review the administrator responsibilities and let me know if you'd like me add or change anything. Thank you.

### CODE REVIEW

**Comment by inoxx03** [comment]
@stoobie Thank you for the feedback on the draft, I have incorporated your suggestions into this PR. please feel free to let me know if you'd like me to make other changes.

**Comment by inoxx03** [comment]
@cbudz @redhat-documentation/ccs-style-council Please let me know your thoughts  and suggestions about the admin responsibility list. If you think tit is OK, please feel free to approve this PR.

Thank you.

**Comment by inoxx03** [comment]
Thank you for the edits @leswilliams44 :)

File: ADMIN.adoc
Comment (stoobie) [inline ADMIN.adoc:30]: * Review open link:https://github.com/redhat-documentation/supplementary-style-guide/pulls[pull requests].

File: ADMIN.adoc
Comment (stoobie) [inline ADMIN.adoc:10]:
```suggestion
As the administrator, you are the point of contact if any contributors experience issues with the repository.

In addition, you are in charge of the following:
* Maintaining the Style Guide site on GitHub
* Driving resolution of open issues
* Driving completion of pull requests
```

File: ADMIN.adoc
Comment (stoobie) [inline ADMIN.adoc:22]: I don't think there should be links in headings. You already have this link in the first bullet.

In general, I think that having duplicate links in the same topic is not user-friendly.

**Review by stoobie — DISMISSED** [review]
@inoxx03 Looks good overall! I just had a few pedantic comments.

Thank you so much for putting together this PR!

File: ADMIN.adoc
Comment (ndeevy) [inline ADMIN.adoc:12]: Maybe point of contact should be lowercase?

**Review by ndeevy — APPROVED** [review]

**Comment by ndeevy** [comment]
> @stoobie @cbudz @bergerhoffer @redhat-documentation/ccs-style-council Please feel free to review the administrator responsibilities and let me know if you'd like me add or change anything. Thank you.

LGTM!  I left one small comment :)

File: ADMIN.adoc
Comment (bergerhoffer) [inline ADMIN.adoc:26]: I would probably remove this, I think this might confuse people more than help. We don't really have priority labels (like low, medium, high). The only thing we have really is the "on hold", which I don't think we need admin rotation members to worry about adding.

**Review by bergerhoffer — COMMENTED** [review]
@inoxx03 Thanks a lot for drafting these responsibilities! Just a few minor comments.

What do you think about including a section to this page from the README page? Otherwise people might not know that it exists.

File: ADMIN.adoc
Comment (JoanneOFlynn2018) [reply ADMIN.adoc:10]: Instead of "In addition, you are in charge of the following:" in @stoobie's suggested change, how about "You also have the following responsibilities:"

File: ADMIN.adoc
Comment (JoanneOFlynn2018) [inline ADMIN.adoc:14]:
```suggestion
* You are the point of contact if contributors experience issues with the repository.
```

Comment (JoanneOFlynn2018) [reply ADMIN.adoc:14]: Being pedantic now rather than someone pointing out these minor issues down the line.

File: ADMIN.adoc
Comment (JoanneOFlynn2018) [reply ADMIN.adoc:22]: +1

File: ADMIN.adoc
Comment (cbudz) [reply ADMIN.adoc:12]: +1

File: ADMIN.adoc
Comment (cbudz) [reply ADMIN.adoc:17]: I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

File: ADMIN.adoc
Comment (cbudz) [reply ADMIN.adoc:26]: +1.

File: ADMIN.adoc
Comment (cbudz) [reply ADMIN.adoc:35]: @bergerhoffer @inoxx03 Style Council is also just an informal name that doesn't have any defined meaning outside of CCS, so maybe just referring to "project members" is best across this doc.

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:27]: @stoobie @JoanneOFlynn2018 Got it, I moved the links form headings to regular text and I removed the duplicates.

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:17]: It should:) Fixed. Thank you:)

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:25]: Good point! Thank you. Changed

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:31]: @bergerhoffer That's reasonable. Thank you. I removed the line per your suggestion:)

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:22]: > @inoxx03 Thanks a lot for drafting these responsibilities! Just a few minor comments.
> 
> What do you think about including a section to this page from the README page? Otherwise people might not know that it exists.

@bergerhoffer I think that is a good point. Let me add the link after I incorporate the collective feedback form project members. Thank you.

File: ADMIN.adoc
Comment (inoxx03) [reply ADMIN.adoc:40]: @cbudz @bergerhoffer Sounds good. Replaced references to _CCS style council_ with the more generically-worded term _project members_.

File: ADMIN.adoc
Comment (inoxx03) [inline ADMIN.adoc:21]: > I would also recommend updating the instruction to flag the issue with "Needs Fix" (we can add that if people agree) and simply @ the admin group (redhat-documentation/ccs-style-council)

@cbudz I added a note about the _Needs Fix_ label, but was not sure where you wanted me to place it. Is it OK if I include it under _Driving resolution of open issues_?

**Comment by inoxx03** [comment]
@bergerhoffer @cbudz @stoobie @JoanneOFlynn2018 @ndeevy Thank you for the feedback. I addressed your comments and updated the list. Please let me know what you think.

File: README.md
Comment (inoxx03) [inline README.md:23]: @bergerhoffer I linked to the admin responsibilities list from README.adoc. Please let me know whether this is OK, or you'd like me to change anything about this section. Thank you.

File: README.md
Comment (ndeevy) [inline README.md:19]: Such a tiny thing but it should be 'information about' not 'information on' to be fully IBM-style compliant. Same for the Administrator responsibilities below.
Overall looks great to me though! Thanks @inoxx03

Comment (inoxx03) [reply README.md:19]: @ndeevy Very true:) Thank you. Although, I don't think that I added or changed this line in my current PR, let me fix it anyway:)

Comment (inoxx03) [reply README.md:19]:
```suggestion
See the [Contributing guide](CONTRIBUTING.md) for information about how to contribute an update.
```

File: ADMIN.adoc
Comment (stoobie) [reply ADMIN.adoc:21]: @inoxx03 That does look like a logical place to me.

**Review by ndeevy — DISMISSED** [review]

File: ADMIN.adoc
Comment (cbudz) [inline ADMIN.adoc:8]: @inoxx03 please insert at the end of this section that each administration rotation is marked on the Style Guide shared calendar, and admins will receive a notification and calendar invite when it's their turn.

File: ADMIN.adoc
Comment (cbudz) [reply ADMIN.adoc:21]: @inoxx03 that works for me.

File: ADMIN.adoc
Comment (cbudz) [inline ADMIN.adoc:23]: suggestion: "include a link to the PR that resolves the issue, if one exists."

**Review by cbudz — COMMENTED** [review]
Added a few comments/suggestions here @inoxx03

File: ADMIN.adoc
Comment (bergerhoffer) [inline ADMIN.adoc:7]: Would you mind making these links to https://github.com/cbudz and https://github.com/bergerhoffer?

File: ADMIN.adoc
Comment (bergerhoffer) [inline ADMIN.adoc:21]: Maybe make @redhat-documentation/ccs-style-council a link to https://github.com/orgs/redhat-documentation/teams/ccs-style-council?

Comment (bergerhoffer) [reply ADMIN.adoc:21]: Also, what is the "Needs fix" label for? I see that we don't have this label and I'm not sure I understand its purpose here. CC @cbudz

**Review by bergerhoffer — COMMENTED** [review]
Appreciate the updates, just a few more comments.

**Comment by bergerhoffer** [comment]
@inoxx03 Can you take a look at these last few comments? Then I think this should be ready to go!

File: ADMIN.adoc
Comment (bergerhoffer) [reply ADMIN.adoc:21]: After discussing, the Needs fix label isn't needed. CC @inoxx03

Comment (inoxx03) [reply ADMIN.adoc:21]: Thank you @bergerhoffer. I'll update the PR and remove the line about the _Needs Fix_ label.

**Comment by inoxx03** [comment]
@cbudz @bergerhoffer I've updated the PR and added information to the admin responsibilities checklist based on your suggestions. Please let me know if you'd like me to make any further changes. if the list looks OK to you now, please go ahead and merge this PR. Thank you:)

**Review by gregraka — DISMISSED** [review]
LGTM

**Review by leswilliams44 — APPROVED** [review]

File: ADMIN.adoc
Comment (bergerhoffer) [inline ADMIN.adoc:7]: The second link needs to be updated to be bergerhoffer

**Review by bergerhoffer — COMMENTED** [review]
Just one link that needs to be fixed!

@gregraka Once that link is fixed, you can approve the PR (if the update looks good to you) and you can count all of the previous approvals, including mine as a third, and merge. Thanks!

**Comment by cbudz** [comment]
Adding my "Ack" to the chorus here.

**Review by ndeevy — APPROVED** [review]

File: ADMIN.adoc
Comment (gregraka) [reply ADMIN.adoc:7]: @inoxx03 Would you be able to push a new commit with Andrea's requested change (earlier)?

Thanks, --Greg

Comment (inoxx03) [inline ADMIN.adoc:7]:
```suggestion
To volunteer to be an administrator, contact link:https://github.com/cbudz[@cbudz] or link:https://github.com/bergerhoffer[@bergerhoffer].
```

Comment (inoxx03) [reply ADMIN.adoc]: @gregraka Thank you for spotting this! My bad. Fixed in 6cd6d68
CC @bergerhoffer

**Review by gregraka — APPROVED** [review]
LGTM!

Let's merge this puppy!! :^)

**Pull request merged**
